Commands/CLI/seval.js

```diff
@@ -0,0 +1,21 @@
+module.exports = async ({ cli }, cmdData, args) => {
+	if (!args) return winston.warn("Missing arguments for seval");
+	let [shard, ...code] = args.split(" ");
+	if (isNaN(parseInt(shard))) {
+		code.unshift(shard);
+		shard = 0;
+	} else {
+		shard = parseInt(shard);
+	}
+	let s;
+	if (code.length === 0) return winston.error("Missing code.");
+	if (!cli.sharder.shards.get(shard)) {
```

Comment:
`.has` instead of `.get`

Reply:
There's no point to change it in my opinion. Both can return a falsy value, so 🤔

Reply:
Yes, but has checks just if the value is present, while get also returns it.